kytos/core/auth.py

```diff
@@ -284,7 +289,8 @@ def _authenticate_user(self):
         except TypeError as err:
             raise BadRequest("Credentials were not sent.") from err
         user = self._find_user(username)
-        if user["password"] != hashlib.sha512(password).hexdigest():
+        password_hashed = UserDoc.hashing(password, user["hash"])
+        if user["password"] != password_hashed:
```

Comment:
@Alopalao if an user's `state` isn't `"active"` it shouldn't be allowed to authenticate again. Could you also cover this? Otherwise soft deleted users would be able to login.

Reply:
Awesome. Let me close this thread.